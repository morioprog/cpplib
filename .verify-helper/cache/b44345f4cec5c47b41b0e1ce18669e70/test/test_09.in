8
2 7
-4 7
8 5
-1 1
0 3
5 6
-9 4
-5 3
