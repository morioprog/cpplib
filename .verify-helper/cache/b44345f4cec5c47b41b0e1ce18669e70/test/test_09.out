8 5
5 6
2 7
0 3
-4 7
-1 1
-5 3
-9 4
